tests/network-tests/src/scenarios/content-directory.ts

```diff
@@ -4,7 +4,10 @@ import nftAuctionAndOffers from '../flows/content/nftAuctionAndOffers'
 import { scenario } from '../Scenario'
 
 scenario('Content directory', async ({ job }) => {
-  const leadSetupJob = job('Set content working group leads', leadOpening(true, ['contentWorkingGroup']))
+  const leadSetupJob = job(
+    'Set content working group leads',
+    leadOpening(true, ['contentWorkingGroup', 'storageWorkingGroup'])
+  )
```

Comment:
This change is not needed after changes from https://github.com/Joystream/joystream/pull/3421 are merged into rhodes. Storage node, as well as storage working group leader, will be available. Please revert this change; you can merge #3421 into this branch to make it work again.